### DISCUSSION TITLE
Frigate+ Snapshots Scroll Limit

### DISCUSSION THREAD

**Discussion by ScottulusMaximus** [blakeblackshear/frigate#14425] [General]
When reviewing the pictures to do the "this is/this is not" on the Frigate+ tab you can only scroll down 60 lines, it won't load past that.

The files are there, when sorting by reverse date it'll pick up the images from the set retention. Also filtering cameras and scores you can go further back but it always caps at 60 lines.

I've got 9 cameras and get a LOT of detections so would be nice to be able to load more making it easier to upload training data.

**Comment by NickM-27** [comment]
Infinite scrolling is already supported, if it's not working then that would be a bug but in any case it'll already be fixed for the next version.
